src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const th = table.tHead.rows[0].cells[1];
+const targetTh = table.tHead.rows[0].cells[4];
+const newTh = table.tHead.rows[0].insertCell(5);
+
+newTh.outerHTML = targetTh.outerHTML;
+targetTh.innerHTML = th.innerHTML;
+
+const tf = table.tFoot.rows[0].cells[1];
```

Comment:
Again, using hardcoded indices like `cells[1]` can lead to issues if the table structure changes. Consider a more flexible approach.